src/components/Board.js

```diff
@@ -15,11 +16,85 @@ class Board extends Component {
       cards: [],
     };
   }
+  componentDidMount = () => {
+    console.log('Component did mount');
+    axios.get('https://inspiration-board.herokuapp.com/boards/Aruna/cards')
+    .then((response) => {
+      console.log(response.data);
+      this.setState({
+        cards: response.data
+      });
+
+    })
+    .catch((error) => {
+      this.setState({
+        error: error.message
+      })
+
+    });
+  }
+  removeCard = (id) => {
+    let cards = this.state.cards;
+    axios.delete(`https://inspiration-board.herokuapp.com/boards/:board_name/cards/${id}`)
+    .then((response) => {
+      cards = cards.filter(card =>card.card.id!=id);
+      this.setState({
+        cards,
+        error:'Card was deleted successfully',
+      });
+    })
+    .catch( (error) => {
+      this.setState({ error: error.message });
+    });
+  }
+  renderCards = () => {
+    const componentList = this.state.cards.map((card,index) => {
+      return(
+        <Card
+          key={index}
+          id={card.card.id}
+          text={card.card.text}
+          emoji={card.card.emoji}
+          removeCard={this.removeCard}
+          />
+      );
+    });
+    return componentList;
+  }
+
+  addCard = (card) => {
+    const cards = this.state.cards;
+    const newCard = {'card': card};
+  axios.post('https://inspiration-board.herokuapp.com/boards/Aruna/cards', card)
+      .then((response) => {
+        cards.push(newCard);
+        this.setState({
+          cards,
```

Comment:
You're adding a card without an ID.  You'll need to get an id value from the `response`.